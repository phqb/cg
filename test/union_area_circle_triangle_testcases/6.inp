0 0 1
-0.24 1.36
-1.11 -0.77
0.6 -0.19