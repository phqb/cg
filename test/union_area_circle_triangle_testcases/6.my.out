1.4357289934